x-pack/plugins/infra/public/components/asset_details/hooks/use_data_views_provider.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
```

Comment:
Do we need this hook AND useDataView?  Seems like we could replace useDataView with this one.  is the useDataView provider only wrapping the host listing page now?

Reply:
I wanted to keep them separate because the host view doesn't need the logs data view and the logs view reference configuration is slightly different because it adds `host.name` as an extra field.